Composer/packages/client/src/pages/design/exportSkillModal/content/SelectTriggers.tsx

```diff
@@ -83,6 +83,7 @@ export const SelectTriggers: React.FC<ContentProps> = ({ setSelectedTriggers, pr
   const selection = useMemo(
```

Comment:
same comments as the other selection in the other file

Reply:
same as above